mark-jacob/starter-code/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Lab 1</title>
+  <!-- DONE: Add a meta tag to properly set the viewport and initial scale -->
+  <meta name="viewport" content="width=device-width">
```

Comment:
this works, but you also need to set initial scale

`<meta name="viewport" content="width=device-width, initial-scale=1">`